LAMPAPI/contactController.php

```diff
@@ -96,6 +96,8 @@ function updateContact() {
         ErrorHandler::generic_error(new Error("Contacts could not be updated", "Please try again later."));
     }
 
+    echo json_encode($result);
+
```

Comment:
We wouldn't need to output a response because we can refer to the HTTP error code if something went wrong.

Reply:
Ah yes, that is just leftover from some debuggging, will take out now!